nautobot/dcim/forms.py

```diff
@@ -2285,6 +2286,11 @@ class DeviceFilterForm(
         label="Has rear ports",
         widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
     )
+    is_controller = forms.NullBooleanField(
+        required=False,
+        label="Is a controller",
+        widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES),
+    )
```

Comment:
Was this getting the wrong widget by default?

Reply:
I thought I removed this code as I removed the filter. I am not sure we need this as a filter or not.  But, this is common for all of our NullBooleanField to have the StaticSelect2 widget.

Reply:
I did remove this, I think you are looking at an old commit.